ENIGMAsystem/SHELL/Audio_Systems/DirectSound/DSbasic.cpp

```diff
@@ -1,4 +1,4 @@
-/** Copyright (C) 2013 Robert B. Colton
+/** Copyright (C) 2013, 2018 Robert B. Colton
```

Comment:
Amazing how time flies, mm?

Reply:
This comment just makes me want to shed tears 😿